### DISCUSSION TITLE
Node version

### DISCUSSION THREAD

**Discussion by Suhaas10** [vercel/community#1256] [Help] [answered]
### Summary

I'm unable to deploy on vercel. Last deployment was around 2 months ago. Today I tried to deployed again. Getting this error

        The engine "node" is incompatible with this module. Expected version "14.20.0". Got "14.21.1"
        
I have specified the node version as 14.20.0 in package.json. I'm not sure from where/why vercel is picking 14.21.1
I haven't changed any dependencies recently.

**Comment by amyegan** [comment, marked as answer]
Hi @Suhaas10! Vercel automatically rolls out minor and patch updates for major Node versions as needed to address stability and security issues (more info in [runtime docs](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version)). If you change the `package.json` to specify just the major version `14`, the build should succeed.

**Reply by Suhaas10** [reply]
Thanks a lot @amyegan! Worked for me
